slices/libnss3.yaml

```diff
@@ -1,18 +1,39 @@
+# Network Security Service libraries.
+#
+# This is a set of libraries designed to support cross-platform development of
+# security-enabled client and server applications. It can support SSLv2 and v4,
+# TLS, PKCS #5, #7, #11, #12, S/MIME, X.509 v3 certificates and other security
+# standards.
 package: libnss3
 
 essential:
   - libnss3_copyright
 
 slices:
+  # The libs slice contains all the libraries coming from this package.
   libs:
+    essential:
+      # This package still has many other libraries which are not part of any
+      # slices yet. Upon creation of new slices containing those libraries,
+      # make sure to add those slices in this list.
+      - libnss3_nss
+      - libnss3_smime
+
+  nss:
     essential:
       - libc6_libs
       - libnspr4_libs
-      - libsqlite3-0_libs
```

Comment:
Turns out we never needed `libsqlite3-0_libs` for the `libnss3.so` and `libnssutil3.so` files.

Reply:
Turns out I was wrong :facepalm: Added the slice back. Thanks @vpa1977.